Marlin/language_de.h

```diff
@@ -90,29 +90,29 @@
 #define MSG_PID_I                           "PID I"
 #define MSG_PID_D                           "PID D"
 #define MSG_PID_C                           "PID C"
-#define MSG_ACC                             "A"
-#define MSG_VXY_JERK                        "V xy Ruck"
-#define MSG_VZ_JERK                         "V z  Ruck"
-#define MSG_VE_JERK                         "V e  Ruck"
-#define MSG_VMAX                            "V max " // space by purpose
+#define MSG_ACC                             "Beschleunig."
+#define MSG_VXY_JERK                        "V xy Bew-Mom."
+#define MSG_VZ_JERK                         "V z  Bew-Mom."
+#define MSG_VE_JERK                         "V e  Bew-Mom."
+#define MSG_VMAX                            "V Max " // space by purpose
 #define MSG_X                               "X"
 #define MSG_Y                               "Y"
 #define MSG_Z                               "Z"
 #define MSG_E                               "E"
-#define MSG_VMIN                            "V min"
-#define MSG_VTRAV_MIN                       "VTrav min"
-#define MSG_AMAX                            "A max " // space by purpose
-#define MSG_A_RETRACT                       "A Rückzug"
-#define MSG_A_TRAVEL                        "A Reise"
-#define MSG_XSTEPS                          "X steps/mm"
-#define MSG_YSTEPS                          "Y steps/mm"
-#define MSG_ZSTEPS                          "Z steps/mm"
-#define MSG_ESTEPS                          "E steps/mm"
+#define MSG_VMIN                            "V Min"
+#define MSG_VTRAV_MIN                       "V Beweg. Min"
+#define MSG_AMAX                            "A Max " // space by purpose
+#define MSG_A_RETRACT                       "A Retract"
+#define MSG_A_TRAVEL                        "A Bewegung"
+#define MSG_XSTEPS                          "X Schr./mm"
+#define MSG_YSTEPS                          "Y Schr./mm"
+#define MSG_ZSTEPS                          "Z Schr./mm"
+#define MSG_ESTEPS                          "E Schr./mm"
```

Comment:
In der Tat. Besonders wenn sie in der Lage waren Marlin zu konfigurieren.
Aber dann gibt es auch immer wieder welche die den Drucker von der Stange kaufen und dann wie der Ochs vorm Berge stehen. Aber auch für die, ist es dann hilfreicher wenn sie nach einem Begriff suchen können, als wenn sie einen im Marlin-Zusammenhang ungebräuchlichen deutschen Begriff vorgesetzt bekommen. 
Aber was wissen wir schon? Die die das hier sinnfoll übersetzen können brauchen die Übersetzung nicht und die, die die Übersetzng brauchen, können sie nicht selbst machen.
Von "Schr." auf "Schritte" zu kommen ist auch nicht trivial wenn man gar keine Ahnung hat um was es gehen könnte.